stetho/src/main/java/com/facebook/stetho/inspector/elements/ChainedDescriptor.java

```diff
@@ -172,6 +172,16 @@ public final Object getChildAt(Object element, int index) {
     return onGetChildAt((E)element, thisIndex);
   }
 
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void setAttributesAsText(Object element, String text) {
```

Comment:
oh ... `setAttributesAsText` and `onSetAttributesAsText` are sandwiched between `getChildAt` and `onGetChildAt`. We need to keep the paired methods next to each other.